Tests/test_PAML_codeml.py

```diff
@@ -472,6 +472,24 @@ def testParsePairwise(self):
             pairwise = results["pairwise"]
             self.assertEqual(len(pairwise), 5, version_msg)
 
+    def testParseSitesParamsForPairwise(self):
+        """Verify that pairwise site estimates are indeed parsed. Fixes #483"""
+        res_dir = os.path.join(self.results_dir, "codeml", "pairwise")
+        for results_file in os.listdir(res_dir):
+            version = results_file.split('-')[1].split('.')[0]
+            version_msg = "Improper parsing for version %s" \
+                        % version.replace('_', '.')
+            results_path = os.path.join(res_dir, results_file)
+            results = codeml.read(results_path)
+            for seq2 in results['pairwise'].values():
```

Comment:
The test doesn't actually catch the problem as written.  Changing 484-485 to this should work:

            for seq in results["pairwise"]:
                seq_vals = results["pairwise"][seq]
                self.assertTrue(len(seq_vals) > 0, "no parsed parameters for '%s'" \
                                % seq)
                for data in seq_vals: